tests/foreman/api/test_errata.py

```diff
@@ -1430,3 +1464,247 @@ def rh_repo_module_manifest(module_sca_manifest_org, module_target_sat):
+    assert len(lce_list) == number_of_lces + 1
+    # colelct default CV for org
+    default_cv = (
+        target_sat.api.ContentView(
+            organization=module_sca_manifest_org,
+            name=DEFAULT_CV,
+        )
+        .search()[0]
+        .read()
+    )
+    cv_list = list([default_cv])
+    # for each environment including 'Library'
+    for _lce in lce_list:
+        # create some new CVs with some content
+        for _i in range(number_of_cvs):
+            new_cv = target_sat.api.ContentView(
+                organization=module_sca_manifest_org,
+                repository=[custom_repo],
+            ).create()
+            # lces to be promoted to, omit newer than _lce in loop
+            env_ids = sorted([lce.id for lce in lce_list if lce.id <= _lce.id])
+            # when the only lce to publish to is Library, pass None to default
+            if len(env_ids) == 1 and env_ids[0] == lce_library.id:
+                env_ids = None
+            # we may initially promote out of order, use force to bypass
+            new_cv = cv_publish_promote(
+                target_sat,
+                module_sca_manifest_org,
+                new_cv,
+                lce=env_ids,
+                force=True,
+            )['content-view']
+            cv_list.append(new_cv)
+
+    # total amount of CVs created matches expected and search results
+    assert len(cv_list) == 1 + (number_of_cvs * (number_of_lces + 1))
+    assert len(cv_list) == len(
+        target_sat.api.ContentView(organization=module_sca_manifest_org).search()
+    )
+    # newest created lce and CV used for host
+    host_lce = lce_list[-1].read()
+    host_cv = cv_list[-1].read()
+    # one ak with host CV and lce
+    ak = target_sat.api.ActivationKey(
+        organization=module_sca_manifest_org,
+        environment=host_lce,
+        content_view=host_cv,
+    ).create()
+    # register contenthost with ak to newest cv lce
+    result = rhel8_contenthost.register(
+        org=module_sca_manifest_org,
+        activation_keys=ak.name,
+        target=target_sat,
+        loc=None,
+    )
+    assert result.status == 0, f'Failed to register the host: {rhel8_contenthost.hostname}'
+    assert rhel8_contenthost.subscribed
+    rhel8_contenthost.execute(r'subscription-manager repos --enable \*')
+    # Installing all outdated packages
+    pkgs = ' '.join(FAKE_9_YUM_OUTDATED_PACKAGES)
+    assert rhel8_contenthost.execute(f'yum install -y {pkgs}').status == 0
+    rhel8_contenthost.execute('subscription-manager repos')
+    # After installing packages, check available incremental updates
+    host = rhel8_contenthost.nailgun_host.read()
+    response = target_sat.api.Host().bulk_available_incremental_updates(
+        data={
+            'organization_id': module_sca_manifest_org.id,
+            'included': {'ids': [host.id]},
+            'errata_ids': FAKE_9_YUM_SECURITY_ERRATUM,
+        },
+    )
+    assert (
+        not response
```

Comment:
Is there a particular reason you are asserting this here? Would it be better to assert what you actually expect to be returned? I'd think you'd know how many updates should be available.

Reply:
right, if there are no incremental updates, response will be just `[]`, I can change to that

Reply:
I believ you have already addressed the @sambible 's comment, I am resolving this conversation.